CONTRIBUTING.md

```diff
@@ -249,6 +244,6 @@ Example:
 
 ```sh
 DRIVER_SPEED=slow \
-             WEB_DRIVER=chrome \
-             DEBUG_DRIVE=true \
-             ./bin/rails runner "MiBridges::Driver.new(snap_application: SnapApplication.find(ID)).run"
+  WEB_DRIVER=chrome \
```

Comment:
you know what else would be cool? to have a `bin/demo_drive` script that does this for us!